autoload/gh/gh.vim

```diff
@@ -229,3 +229,18 @@ function! gh#gh#decide_open() abort
   endif
   return ''
 endfunction
+
+function! gh#gh#get_token() abort
+  let token = get(g:, 'gh_token', '')
+  if !empty(token)
+    return token
+  endif
+  let s:gh_token_path = glob('~/.config/gh/hosts.yml')
```

Comment:
1. Is this works on Windows?
2. I think we need use `expand()` to get full path.

Reply:
I am now trying to test it on github actions 👍

Reply:
@skanehira 
Sorry for this mention.

The script is tested on github actions with vim and Windows. It seems that `glob` can handle all the Unix like paths on windows.
https://github.com/Matts966/gh.vim/actions/runs/520828192

Neovim is not tested currently because it fails in installing and cancel other tests.

I mistook the branch and the all script is on this PR. If it is not desired, I will remove test related commits or squash. Please notify me.

Thank you :)

Reply:
@Matts966
Thanks :)
Could you separate test from this PR?
Because this PR goals is not add test.

I'll merge this PR when separated.

Reply:
I create a issue to add test and CI.
https://github.com/skanehira/gh.vim/issues/80

Reply:
Thank you for issueing :)
I finished 2 pull requests. the latter one for test depends on this PR so based on this branch.

Thank you!

Reply:
Thanks!